pkg/apply/apply.go

```diff
@@ -41,9 +41,7 @@ func UpsertEnvVar(envVars *[]corev1ac.EnvVarApplyConfiguration, envVar ...corev1
 }
 
 func UpsertEnvVars(envVars *[]corev1ac.EnvVarApplyConfiguration, upEnvVars ...corev1ac.EnvVarApplyConfiguration) {
-	for _, e := range upEnvVars {
-		upsert(envVars, e, byEnvVarName)
-	}
+	UpsertEnvVar(envVars, upEnvVars...)
```

Comment:
@akagami-harsh Can we just remove the `UpsertEnvVar()` function and use `UpsertEnvVars()` everywhere ?
Including this part: https://github.com/kubeflow/trainer/blob/9dd455e82228a8f40d6364cfcafc8ddd17f91e77/pkg/runtime/core/trainingruntime.go#L225-L227